test/helpers/user_blocks_helper_test.rb

```diff
@@ -0,0 +1,43 @@
+# coding: utf-8
+require "test_helper"
+include ApplicationHelper
```

Comment:
Yes, because the context there is `Object` so you wind up including the methods in `ApplicationHelper` into every single object in the system ;-)

It's not in `UserBlocksHelper` because that's a module that doesn't include it. But the things which include `UserBlocksHelper` at run time will have included it.

You should do what the browse and notes helper tests do and just include that inside the helper class.